src/components/PostComment.js

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import { postCommentById } from "../api";
+import Loading from "./Loading";
+
+const PostComment = ({ article_id }) => {
+  const [commentInput, setCommentInput] = useState("");
+  const [isLoading, setIsLoading] = useState(false);
+  const [posted, setPosted] = useState(false);
+
+  const handlePostRequest = (event) => {
```

Comment:
`handlePostRequest` isn't the most accurate name here for a function that is designed to handle a change